examples/angular/tools/ngsw_config.bzl

```diff
@@ -11,27 +11,25 @@ def ngsw_config(name, config, index_html, src, out = None, **kwargs):
 
     nodejs_binary(
         name = ngsw_config_name,
-        data = ["@npm//@angular/service-worker", index_html, config, src],
```

Comment:
I think including these here is one thing that was causing an issue, although I don't fully understand why... including the `src` here (output of `ts_project`) was somehow causing the typescript compiler to re-run but without the correct tsconfig that adds the `paths`. So typescript was unnecessary re-compiling with the wrong tsconfig which failed with path errors.